solvcon/solver_legacy.py

```diff
@@ -286,6 +288,10 @@ def bind(self):
         for it in range(self.ncore):
             self.arglists.append([byref(self.exd), c_int(0), c_int(0)])
 
+    def check_blk_pointer(self):
+        msh = solvcon.mesh.Mesh()
+        msh._check_pointers(self.blk)
```

Comment:
You shouldn't instantiation a Mesh object just for calling its staticmethod.  Simply write Mesh._check_pointers(self.blk).